pkg/summary/summarizers.go

```diff
@@ -204,6 +206,22 @@ func initializeCheckErrors() {
 	logrus.Debugf("GVK Error Mapping not provided, using predefined values")
 }
 
+func checkGeneration(obj data.Object, _ []Condition, summary Summary) Summary {
+	if summary.Attributes != nil && summary.Attributes["hasObservedGeneration"] == true {
+		metadataGenerationStr := obj.String("metadata", "generation")
+		observedGenerationStr := obj.String("status", "observedGeneration")
+		generation, errMetadata := strconv.ParseInt(metadataGenerationStr, 10, 64)
+		observed, _ := strconv.ParseInt(observedGenerationStr, 10, 64)
+
+		if errMetadata == nil && observed < generation {
```

Comment:
Should we treat observed > generation to also mean in progress? Or active? (this would happen only if a controller has a bug though)

Reply:
This case will be handled by the kstatus library. since kstatus library checks observed < generation.

Reply:
https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/kstatus/status/generic.go#L73

Reply:
Kstatus uses `!=` . So I would the same here.